### PR TITLE
Cpu percentage

### PR DESCRIPTION
**Pull request by solarnz**

### CODE REVIEW

**Comment by solarnz** [comment]
_bump_

File: packages/cpu/cpuperc.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	c.percentages = make(chan float64)
+
+	go func() {
+		defer close(out)
+		go c.collect()
+		defer func() {
+			c.collecting = false
+		}()
+
+		out <- c.generateOutput(0.0)
```
Comment (denbeigh2000) [inline packages/cpu/cpuperc.go:67]: Seriously? 0.0?

Comment (solarnz) [inline packages/cpu/cpuperc.go:67]: Yea, unless you want to default to 100%?

Comment (denbeigh2000) [inline packages/cpu/cpuperc.go:67]: Wait damn I see why you did that now.

**Pull request merged**